src/modules/toolkits/cuda.nix

```diff
@@ -0,0 +1,33 @@
+{ pkgs, lib, config, ... }:
+
+let
+  cfg = config.toolkits.cuda;
+in
+{
+  options.toolkits.cuda = {
+    enable = lib.mkEnableOption "CUDA toolkit";
+
+    package = lib.mkOption {
+      type = lib.types.package;
```

Comment:
Sometimes CUDA can't come from Nix, so we'll have to allow a way to setup FHS in those cases.

It's tricky to get this right (and won't work on macOS), but it's often required.

Reply:
> Sometimes CUDA can't come from Nix,

Interesting. Any specific examples in mind?